pkg/roachpb/gen/BUILD.bazel

```diff
@@ -11,5 +11,6 @@ go_library(
 go_binary(
     name = "gen",
     embed = [":gen_lib"],
+    gotags = ["metamorphic_disable"],
```

Comment:
Due to the way `gotags` works, this change would require re-compiling all the stuff that `pkg/roachpb/gen` and `execgen` depend on including the entire Go standard library with the new `gotags`. I suspect this would have a bigger performance impact than we want considering how small this conditional branch is. 

I would expect instead that we set the environment variable wherever it needs to be set -- i.e. in the `env` attribute of the call to `ctx.actions.run()` in `pkg/roachpb/gen.bzl` and in the various places we call into `execgen` as part of the build. Is there a reason we haven't done that?

Reply:
We went over this in DM and I don't think this is anything to be concerned about for now, we can follow-up on it if it turns out to be relevant.